packages/proto/src/messages/distribution/distribution.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  MsgWithdrawDelegatorReward,
+  MsgWithdrawValidatorCommission,
+  MsgSetWithdrawAddress,
+} from '../../proto/cosmos/distribution/tx'
+
+export interface MsgWithdrawDelegatorRewardProtoInterface {
+  path: string
+  message: MsgWithdrawDelegatorReward
+}
+
+export function createMsgWithdrawDelegatorReward(
+  delegatorAddress: string,
+  validatorAddress: string,
+) {
```

Comment:
this file is not very readable because of all the `exports`, can we group them?

Reply:
@hanchon What exactly are `MsgWithdrawValidatorCommissionProtoInterface` and `MsgSetWithdrawAddressProtoInterface` used for? They're not imported elsewhere in the codebase. Can we delete them?

Reply:
No, the functions are being used by evmos.me to generate that transaction, @fedekunze asked for that transaction some time ago so I just added the helpers here to make a quick release